### ISSUE TITLE
Two issues (Fullscreen/Mouse focus, Changing size of canvas)

### ISSUE THREAD

**Issue by Rogerjwilkinson** [caiiiycuk/js-dos.com#35]
Hi! First off thanks for the awesome and easy to use API.

Just got these two issues I was hoping you knew some tricks for ^^

1) Going into fullscreen seems to lock the mouse into the top left corner of the screen, and disables it. Using the default plunkr example, as well as tried 
```
document.getElementById("fullscreen_btn").addEventListener("click", function() {
    dosbox.requestFullScreen();
```
to no luck.

2) The css seems to get ignored for resizing canvas. It works for the loading, but then game always resizes to 640x400 once everything downloaded.

Thanks for any ideas!

**Comment by x-wolverine-x** [comment]
Cavas resize working without any problems for me.

**Comment by caiiiycuk** [comment]
Fixed in 6.22.37, just use method `fullscreen` from command interface.
http://js-dos.com/#js-dos-622-faq-fullscreen
